pkg/awsutils/awsutils_test.go

```diff
@@ -857,19 +858,30 @@ func TestEC2InstanceMetadataCache_cleanUpLeakedENIsInternal(t *testing.T) {
 	defer ctrl.Finish()
 
 	description := eniDescriptionPrefix + "test"
-	result := &ec2.DescribeNetworkInterfacesOutput{
-		NetworkInterfaces: []*ec2.NetworkInterface{{
-			Description: &description,
-			TagSet: []*ec2.Tag{
-				{Key: aws.String(eniNodeTagKey), Value: aws.String("test-value")},
-			},
-		}},
-	}
+	interfaces := []*ec2.NetworkInterface{{
+		Description: &description,
+		TagSet: []*ec2.Tag{
+			{Key: aws.String(eniNodeTagKey), Value: aws.String("test-value")},
+		},
+	}}
 
-	mockEC2.EXPECT().DescribeNetworkInterfacesWithContext(gomock.Any(), gomock.Any(), gomock.Any()).Return(result, nil)
+	helpCallMockedPagedDescribeNetworkInterface(t, mockEC2, interfaces, nil, 1)
```

Comment:
I would have named it differently - /s/help/setupDescribeNwInterfacesPagesMock...()

Reply:
Will be update soon.